src/visitable.h

```diff
@@ -44,6 +44,13 @@ class visitable {
     item * find_parent( item& it );
     const item * find_parent( const item& it ) const;
 
+    /**
+     * Returns vector of parent containers (if any) starting with the innermost
+     * @param it item to search for which must be contained (at any depth) by this object
+     */
+    std::vector<item *> parents( item& it );
```

Comment:
Nitpicking: technically `it` should be a const reference, the function is not supposed to change the item. Currently I'm a bit surprised by that non-const reference as I have to assume the function will somehow change the item and I can't think any reason for that.